tests/openwisp2/sample_users/migrations/0004_auto_20210118_1211.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 3.1.5 on 2021-01-18 15:11
+
+import phonenumber_field.modelfields
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('sample_users', '0003_update_user_fields'),
+    ]
+
+    operations = [
+        migrations.RemoveField(model_name='user', name='birth_date',),
+        migrations.AlterField(
+            model_name='user',
+            name='location',
+            field=models.CharField(blank=True, max_length=128, verbose_name='location'),
+        ),
+        migrations.AlterField(
+            model_name='user',
+            name='phone_number',
+            field=phonenumber_field.modelfields.PhoneNumberField(
+                blank=True, max_length=128, null=True, region=None, unique=True
+            ),
+        ),
+    ]
```

Comment:
are you sure this migration file is necessary? It shouldn't be.

Reply:
Thanks,
yes this should not be added. I don't understand why it was generated in first place.